ServiceNow/JFrog Spoke/README.md

```diff
@@ -1,8 +1,8 @@
 # Welcome to the JFrog Spoke Documentation
 
-The JFrog Xray Spoke allows your organization to build automated workflows that interact directly with the JFrog Platform.
+The JFrog Spoke allows your organization to build automated workflows that interact directly with the JFrog Platform.
 
-Focused on Xray-related actions: from generation violations reports to creating ignore rules, re-scanning builds and artifacts, assigning custom item properties, and assigning permissions to users and groups, the JFrog Xray Spoke provides out-of-the-box actions that your company can mix-and-match with other ServiceNow workflows to automate your overall IT-operations.
+Focused on Xray-related actions: from generation violations reports to creating ignore rules, re-scanning builds and artifacts, assigning custom item properties, and assigning permissions to users and groups, the JFrog Spoke provides out-of-the-box actions that your company can mix-and-match with other ServiceNow workflows to automate your overall IT-operations.
```

Comment:
Should this block be also reworded to remove "Focussed on Xray ..": or replaced with something like Summary or Functionality or some other heading ?

Reply:
I agree, I am talking to some marketing folks right now on what we should say/focus on.